docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -862,9 +862,10 @@ This endpoint returns a live stream of a container's resource usage statistics.
                "total_usage" : 36488948,
                "usage_in_kernelmode" : 20000000
             },
-            "system_cpu_usage" : 20091722000000000,
+            "system_cpu_usage" : 20091722000000000,
```

Comment:
Is there some weird tab/space thing going on on this like? I don't see any change. Maybe github viewer is on the fritz.